openedx/core/djangoapps/safe_sessions/tests/test_utils.py

```diff
@@ -118,14 +130,20 @@ def assert_request_user_mismatch(self, user_at_request, user_at_response):
             yield
 
     @contextmanager
-    def assert_session_user_mismatch(self, user_at_request, user_in_session):
-        """
-        Asserts that the logger was called when request.user at request
-        time doesn't match the request.user at response time.
-        """
-        with self.assert_logged(
-            "SafeCookieData user at request '{}' does not match user in session: '{}'".format(
-                user_at_request, user_in_session
-            ),
+    def assert_request_and_session_user_mismatch(self, user_at_request, user):
```

Comment:
By looking at their [usage](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/safe_sessions/tests/test_middleware.py#L193-L197) (on master branch), I think the [both](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/safe_sessions/tests/test_utils.py#L106-L131) are serving for same purpose now (their DOC strings were also identical). Before, these context managers were needed to mock logs on different level when one of the logs was warning and other one was error level. Now, both logs are the warnings level so their calls will be in same mock since we are patching onto warning method. So, no need for nested managers like [this](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/safe_sessions/tests/test_middleware.py#L193-L197).